### DISCUSSION TITLE
Can't run examples

### DISCUSSION THREAD

**Discussion by theworkstim** [mistic100/Photo-Sphere-Viewer#981] [General]
Hi apologies I am new to using typescript and npm but have setup and built other projects that work as they should. I am trying to "tweak" one of the components (overlay) within the Photo-Sphere-Viewer code, in order to do that have forked the Git repository and found the src files I need to tweak. I now need to compile the project so I can test it. I have tried various npm commands without success

`npm run `build`` successfully compiles each individual package but I can't see how that is bundled into a top level "dist" folder that the examples are looking for? Any help would be appreciated.

Thanks

Tim

**Comment by mistic100** [comment]
The bundled files are not in the top level dist folder but in the dist folder of each package. The examples "dist" path refers to a mapping on the dev server.

https://github.com/mistic100/Photo-Sphere-Viewer/blob/main/build/liveserver.mjs#L27

**Reply by mistic100** [reply]
Also to run the examples you need to use "npm run serve", it wont work without the custom dev server.

When in doubt, check the scripts section of the package file https://github.com/mistic100/Photo-Sphere-Viewer/blob/main/package.json
